libraries/AP_Quicktune/AP_Quicktune.h

```diff
@@ -0,0 +1,158 @@
+#pragma once
+
+#include <AP_HAL/AP_HAL_Boards.h>
+
+#ifndef AP_QUICKTUNE_ENABLED
+ #define AP_QUICKTUNE_ENABLED 1
+#endif
+
+#if AP_QUICKTUNE_ENABLED
+
+#include <AC_AttitudeControl/AC_AttitudeControl.h>
+#include <AP_Param/AP_Param.h>
+#include <RC_Channel/RC_Channel.h>
+
+class AP_Quicktune {
+public:
+    AP_Quicktune()
+    {
+        AP_Param::setup_object_defaults(this, var_info);
+    }
+
+    // Empty destructor to suppress compiler warning
+    virtual ~AP_Quicktune() {}
+
+    /* Do not allow copies */
+    CLASS_NO_COPY(AP_Quicktune);
+
+    // Parameter block
+    static const struct AP_Param::GroupInfo var_info[];
+
+    void update(bool mode_supports_quicktune);
+    void update_switch_pos(const RC_Channel::AuxSwitchPos ch_flag);
+
+private:
+
+    // Parameters
+    AP_Float enable;
```

Comment:
enable params need to be AP_Int8, or they don't work (which is why QUIK_ params are showing up with QUIK_ENABLE=0)

Reply:
Done.